tidy3d/components/medium.py

```diff
@@ -143,11 +143,52 @@ class NonlinearSusceptibility(NonlinearSpec):
     """
 
     chi3: float = pd.Field(
-        ..., title="Chi3", description="Chi3 nonlinear susceptibility.", units="um^2 / V^2"
+        0, title="Chi3", description="Chi3 nonlinear susceptibility.", units="um^2 / V^2"
     )
 
 
-NonlinearSpecType = Union[NonlinearSusceptibility]
+class KerrAndTPA(NonlinearSpec):
+    """Specification adding an instantaneous nonlinear susceptibility to a medium,
+    as well as a non-instantaneous nonlinear susceptibility representing two-photon
+    absorption (TPA).
+
+    Note
+    ----
+    The nonlinear constitutive relation is solved iteratively; it may not converge
+    for strong nonlinearities. Increasing `numiters` can help with convergence.
+
+    Note
+    ----
+    For complex fields (e.g. when using Bloch boundary conditions), the nonlinearity
+    is applied separately to the real and imaginary parts, so that the above equation
+    holds when both E and :math:`P_{NL}` are replaced by their real or imaginary parts.
+    The nonlinearity is only applied to the real-valued fields since they are the
+    physical fields.
```

Comment:
I'm actually confused by this note (I know that it's just copied from NonlinearSusceptibility...)

The first sentence says that the nonlinearity is applied separately to the real and imaginary parts; but the last sentence *seems* to say that the nonlinearity is only applied to the real fields. Are the "real-valued" fields referenced in the last sentence just each of Re(E) and Im(E), separately? If so can it be reworded to avoid that confusion - or maybe completely removed since the first sentence is already clear. Maybe something like "The nonlinearity is applied to the real/imaginary components separately since each of those represents physical fields."

Reply:
You're right, this is confusing as it stands. I'll change the wording for all such classes

Reply:
yeah, it's each of the real/imaginary components, separately.